pkg/projects/resource_project.go

```diff
@@ -177,22 +198,38 @@ func projectResource() *schema.Resource {
 			})
 		}
 
+		if users != nil {
+			errors = packUsers(d, "user", users)
+		}
+
+		if errors != nil && len(errors) > 0 {
+			return diag.Errorf("failed to pack project %q", errors)
+		}
+
 		return nil
 	}
 
 	var readProject = func(ctx context.Context, data *schema.ResourceData, m interface{}) diag.Diagnostics {
 		project := Project{}
+
 		_, err := m.(*resty.Client).R().SetResult(&project).Get(projectsUrl + data.Id())
+		if err != nil {
+			return diag.FromErr(err)
+		}
 
+		users, err := readUsers(data.Id(), m)
 		if err != nil {
 			return diag.FromErr(err)
 		}
 
-		return packProject(data, &project)
+		return packProject(data, &project, &users)
 	}
 
 	var createProject = func(ctx context.Context, data *schema.ResourceData, m interface{}) diag.Diagnostics {
-		project, key, err := unpackProject(data)
+		log.Printf("[DEBUG] createProject")
```

Comment:
You actually got this working? it goes to the output in TF?

Reply:
Yep, one needs to set an env var to get the output, of course.